packages/gateway/src/middleware/utils.ts

```diff
@@ -1,4 +1,5 @@
 import { Request } from 'express';
+import { Span } from 'opentracing';
```

Comment:
We import opentracing but it is included as a dev dependency in package.json. Is it normal?